server/jetty/src/main/java/io/deephaven/server/jetty/JettyBackedGrpcServer.java

```diff
@@ -93,17 +90,12 @@ public JettyBackedGrpcServer(
         jetty = new Server();
         jetty.addConnector(createConnector(jetty, config));
 
-        final WebAppContext context =
-                new WebAppContext(null, "/", null, null, null, new ErrorPageErrorHandler(), NO_SESSIONS);
-        try {
-            String knownFile = "/ide/index.html";
-            URL ide = JettyBackedGrpcServer.class.getResource(knownFile);
-            Resource jarContents = Resource.newResource(ide.toExternalForm().replace("!" + knownFile, "!/"));
-            context.setBaseResource(ControlledCacheResource.wrap(jarContents));
```

Comment:
It looks like we don't use `io.deephaven.server.jetty.ControlledCacheResource`; is that correct? If so, should we delete it?

Reply:
Good catch, same problem that js-plugins has... I found that the ServletContextHandler also can be given a base resource and will guide DefaultServlet to provide content from there. We still want to share other settings though - it turns out our DefaultServlet configuration was coming from a jetty xml file, webdefault.xml. Now we explicitly configure it (and don't end up with other annotation scanning from the classpath or service loaders).